src/app/services/actions/query-action-creator-util.ts

```diff
@@ -100,12 +100,16 @@ const makeRequest = (httpVerb: string, scopes: string[]): Function => {
 
     if (query.sampleHeaders && query.sampleHeaders.length > 0) {
       query.sampleHeaders.forEach(header => {
-
-        // We are relying on the graph client to set the content type header.
+        // We are relying on the graph client to set the 'application/json' content type header.
```

Comment:
Will setting the `application/json` header break the SDK?

Reply:
Just checked this out. 
The SDK was previously sending the request with multiple `Content-type: application/json` headers if we add them here. That's why we removed the header.

I have now tested. It looks like the latest update fixes this for us too